server/bin/pbench-sync-satellite.sh

```diff
@@ -266,6 +266,13 @@ if [ -s ${state_change_log} ] ;then
     fi
 fi
 
+if [ $nerrs == 0 ] ; then
+    for entry in "${logdir_for_remote}"/*
+    do
```

Comment:
We have typically been keeping the `do` on the same line as the `for` with a semi-colon (`;`) separating them.

Reply:
ok